OWNERS

```diff
@@ -0,0 +1,2 @@
+maintainers:
```

Comment:
Should be "assignees" or "approvers".

For example, see https://github.com/kubernetes/kubernetes/blob/master/docs/OWNERS